### PR TITLE
Add compression savings to pool list page

### PR DESCRIPTION
**Pull request by aruniiird**
This PR is on top of @GowthamShanmugam 's PR#9252, just adding the compression savings details to the list page.

### CODE REVIEW

**Comment by aruniiird** [comment]
@cloudbehl , @afreen23 , @bipuladh  please take a look...

**Comment by GowthamShanmugam** [comment]
Please resolve conflicts and share some screen shot for this

**Review by afreen23 — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-list-page.tsx [hunk 1]
```diff
@@ -120,6 +120,8 @@ const BlockPoolTableRow: React.FC<RowFunctionArgs> = ({ obj, index, key, style,
 
   // Metrics
   const rawCapacity: string = humanizeBinaryBytes(props.poolRawCapacity[name] || 0).string;
+  const compressionSavings: string = humanizeBinaryBytes(props.poolCompressionSavings[name] || 0)
+    .string;
```
[the remark this answers is not in the document]
Comment (aruniiird) [reply frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-list-page.tsx]: Right @afreen23 , updated the code...
@GowthamShanmugam , @afreen23 , screenshot attached, please take a look.

Screenshot from 2021-07-22 20-07-17

**Comment by aruniiird** [comment]
Rebased to latest upstream/master. @afreen23 , @GowthamShanmugam , please take a look

**Review by GowthamShanmugam — APPROVED** [review]

**Comment by GowthamShanmugam** [comment]
Changes are fine, LGTM

**Review by afreen23 — APPROVED** [review]
/lgtm

**Comment by aruniiird** [comment]
Rebased to latest upstream/master.

**Review by GowthamShanmugam — APPROVED** [review]

**Review by afreen23 — APPROVED** [review]
/lgtm

**Pull request merged**